tests/compile_cmake.sh

```diff
@@ -149,13 +147,17 @@ CCPP_CMAKE_FLAGS=$(trim "${CCPP_CMAKE_FLAGS}")
 
 (
   source $PATHTR/NEMS/src/conf/module-setup.sh.inc
-  module use $PATHTR/modulefiles/${MACHINE_ID}
-  module load fv3
+  if [[ $MACHINE_ID == macosx.* ]] || [[ $MACHINE_ID == linux.* ]]; then
+    source $PATHTR/modulefiles/${MACHINE_ID}/fv3
+  else
+    module use $PATHTR/modulefiles/${MACHINE_ID}
+    module load fv3
+  fi
 
   cd ${BUILD_DIR}
 
   cmake ${PATHTR} ${CCPP_CMAKE_FLAGS}
-  make -j ${MAKE_THREADS}
+  make -j ${MAKE_THREADS} VERBOSE=1
```

Comment:
Please remove VERBOSE=1 option. It produces huge log files, which we save in every commit. This is useful for debugging but is not necessary in regular builds. VERBOSE=1 should be also removed from FV3/ccpp/build_ccpp.sh but that's outside the scope of this PR.

Reply:
Done. Let me know if you prefer to remove the  SIONLIB hooks, since they are optional and not needed to run the model. MKL will have to stay for macOS, though.

Reply:
If they are not required then yes, please remove the SIONLIB hooks.

Reply:
Done!